lib/src/jsMain/kotlin/ch/ubique/libs/ktor/common/PathExtensions.js.kt

```diff
@@ -0,0 +1,7 @@
+package ch.ubique.libs.ktor.common
+
+import kotlinx.io.files.Path
+
+internal actual fun Path.freeSpace(): Long {
+	return 1e9.toLong()
```

Comment:
Würde bei allen "nicht"-implementierten Funktionen noch einen Kommentar hinzuschreiben, à la "Kotlin/JS not implemented"